cmsBuild

```diff
@@ -3629,6 +3637,17 @@ def build(opts, args, factory):
             finalString = "\n".join(["%s (%s)" % (dep.pkgName(), dep.checksum)
                                      for dep in getPackages(deps)])
             log(finalString)
+            #get the deps for all packages to be built
+            if opts.depsLog:
```

Comment:
We should always generate the `package.json` even if it has no dependencies. Currently this block of code is under `if pkg.fullDependencies:` condition so for packages with no deps your new code will not run. Move your code outside the condition,.